pkg/api/meta/restmapper.go

```diff
@@ -335,3 +351,13 @@ func (m MultiRESTMapper) AliasesForResource(alias string) (aliases []string, ok
 	}
 	return nil, false
 }
+
+// ResourceInGroup verifies if the group has the resource in it
+func (m MultiRESTMapper) ResourceInGroup(group, resource string) bool {
+	for _, t := range m {
```

Comment:
Maybe we can just add a function called Group() to the RESTMapper interface. The DefaultRESTMapper can implement this by returning its group field. But such a Group() fucntion doesn't make sense for the MultiRESTMapper.